apps/site/astro.config.ts

```diff
@@ -36,20 +36,29 @@ export default defineConfig({
 			customCss: ['./src/styles/starlight.css'],
 			favicon: '/favicon.png',
 			components: {
-				Header: './src/components/StarlightHeader.astro',
+				Header: './src/components/starlight/StarlightHeader.astro',
+				Sidebar: './src/components/starlight/StarlightSidebar.astro',
 			},
 		}),
 		tailwind({
 			applyBaseStyles: false,
 		}),
 		icon({
 			include: {
-				ri: ['github-line', 'twitter-x-line', 'discord-line'],
+				ri: [
+					'github-line',
+					'twitter-x-line',
+					'discord-line',
+					'github-fill',
+					'twitter-x-fill',
+					'discord-fill',
+					'global-fill',
+				],
```

Comment:
```suggestion
				ri: [
					'github-fill',
					'twitter-x-fill',
					'discord-fill',
					'global-fill',
				],
```

Reply:
I forgot that! That was to keep things working while migrating components